eth/hardhat.config.js

```diff
@@ -119,15 +120,21 @@ var config = {
   },
 };
 
+var SEPOLIA_ACCOUNT;
```

Comment:
Nice, I added this in parallel as well to https://github.com/Cardinal-Cryptography/most/pull/248, withou seeing this PR first :)

Reply:
:+1: 
It's going to be fun to merge now :)